src/os/source/source.js

```diff
@@ -251,7 +252,9 @@ os.source.getExportFields = function(source, opt_internal) {
         if (column['visible'] &&
             !goog.string.isEmptyOrWhitespace(goog.string.makeSafe(field)) &&
             !ol.array.includes(fields, field) &&
-            (opt_internal || !os.feature.isInternalField(field))) {
+            (opt_internal || !os.feature.isInternalField(field) ||
+            (opt_includeTime && field == os.data.RecordField.TIME))) {
+              // Todo: `os.data.RecordField.TIME == 'recordTime'` and is written as such in the column
```

Comment:
What outstanding work is there for this todo?

Reply:
Oh, I forgot to clean this up after taking in your suggested code.